CRM/Financeextras/Form/Contribute/CreditNoteEmail.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+use Civi\Api4\CreditNote;
+use Civi\Token\TokenProcessor;
+use Civi\Financeextras\Event\CreditNoteMailedEvent;
+
+/**
+ * Handles Credit Note Email Invoice Task.
+ */
+class CRM_Financeextras_Form_Contribute_CreditNoteEmail extends CRM_Core_Form {
+  use CRM_Contact_Form_Task_EmailTrait;
+
+  /**
+   * The array that holds all the contact ids.
+   *
+   * @var array
+   */
+  public $_contactIds; // phpcs:ignore
```

Comment:
Why do property need to be public? And shouldn't we use `_ `in the variable name? 
 I think it is also better to encapsulate it.

Reply:
Thanks, @erawat, I've changed the property visibility to private, but I have to keep the variable name as it is because it is referenced in the trait here https://github.com/civicrm/civicrm-core/blob/39ca5a727c024da8ff674a5b84aa462dd972baf2/CRM/Contact/Form/Task/EmailTrait.php#L157.

same as the $_context property